src/ch/imedias/rsccfx/view/RsccHomeView.java

```diff
@@ -33,6 +38,17 @@
   ImageView requestImgView;
   ImageView supportImgView;
 
+  final HBox requestBox = new HBox();
+  final VBox requestBoxLabels = new VBox();
+  final HBox supportBox = new HBox();
+  final VBox supportBoxLabels = new VBox();
+
+  Label requestTextBig;
```

Comment:
please change the variable names according to "Field name differentiation" here https://www.cs.technik.fhnw.ch/confluence16/display/VTDESGB/SAD
for example "requestBigLbl"